dags/pegs-challenge-submission-dag.py

```diff
@@ -1,26 +1,35 @@
+import uuid
+import os
 from datetime import datetime
+import pandas as pd
 
 from airflow.decorators import dag, task
 from airflow.models import Param
+from airflow.providers.amazon.aws.hooks.s3 import S3Hook
 
 from orca.services.nextflowtower import NextflowTowerHook
 from orca.services.nextflowtower.models import LaunchInfo
 from orca.services.synapse import SynapseHook
 
+REGION_NAME = "us-east-1"
+BUCKET_NAME = "pegs-challenge-project-tower-scratch"
+FILE_NAME = f"submissions_{uuid.uuid4()}.csv"
+KEY = "work"
 
 dag_params = {
     "synapse_conn_id": Param("SYNAPSE_ORCA_SERVICE_ACCOUNT_CONN", type="string"),
-    "synapse_evaluation_id": Param("9615531", type="string"),
+    "synapse_evaluation_id": Param("9615511", type="string"),
+    "aws_conn_id": Param("AWS_TOWER_PROD_S3_CONN", type="string"),
     "tower_conn_id": Param("PEGS_CHALLENGE_PROJECT_TOWER_CONN", type="string"),
     "tower_run_name": Param("pegs_model_submission_evaluation", type="string"),
```

Comment:
Ah, sorry for the bad info yesterday. I also use `UUID` [here](https://github.com/Sage-Bionetworks-Workflows/orca-recipes/blob/e6048d0989a6d413cc52f63a9d5f243466ebfc1b/dags/dynamic-challenge-dag.py#L26) so that each run has a unique run name which matches its manifest. That's why I had it set as a constant, I was actually using it twice.